presto-docs/src/main/sphinx/optimizer/reorder-joins.rst

```diff
@@ -0,0 +1,30 @@
+*************
+Reorder joins
+*************
+
+Cross joins elimination
+-----------------------
+
+
+When config property ``reorder-joins`` (or ``reorder_joins`` session property) is enabled optimizer will search for cross joins
+in query plan and try to eliminate them by changing order of the joins. While doing so optimizer will try to preserve
+original join order as much as possible.
```

Comment:
Maybe clarify what if no reordering is possible to avoid cross join(s). Would the order of joins change at all?